invokeai/frontend/web/src/features/gallery/store/gallerySlice.ts

```diff
@@ -20,6 +20,7 @@ export const initialGalleryState: GalleryState = {
   boardSearchText: '',
   limit: INITIAL_IMAGE_LIMIT,
   offset: 0,
+  uploadedImages: [],
```

Comment:
It looks like we are manipulating this array but never actually using it anywhere